rootfs/etc/teleport/teleport-auth.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+gomplate -f /etc/teleport/teleport-auth.yaml
+gomplate -f /etc/teleport/teleport-github-connector.yaml
+
+teleport start --config=/etc/teleport/teleport-auth.yaml --roles=auth
```

Comment:
This won't work. It will run in the foreground (as it should). That way k8s can be the supervisor. Please revert to our original logic and use the `postStart` hook for the loop.